ESP8266/ESP8266.cpp

```diff
@@ -279,28 +287,24 @@ bool ESP8266::close(int id)
 
 void ESP8266::setTimeout(uint32_t timeout_ms)
 {
-    _parser.setTimeout(timeout_ms);
+    _parser->setTimeout(timeout_ms);
 }
 
-bool ESP8266::readable()
-{
-    return _serial.readable();
```

Comment:
What should the readable function even mean for a network interface? Doesn't seem to make sense to me as a public API.